### DISCUSSION TITLE
could chainlink organize a 'noob' hackathon ... yes it already does

### DISCUSSION THREAD

**Discussion by n4n0b1t3** [smartcontractkit/full-blockchain-solidity-course-py#1016] [Ideas]
... where the only entry barrier would be the completion of this tutorial?

In the spirit of @cromewar 's answer to my post https://github.com/smartcontractkit/full-blockchain-solidity-course-py/discussions/997

The idea is to get your hands dirty in a kind of safe and friendly environment like this forum here, where it is totally OK to not be a blockchain geek and to be rather on the noob side of the skill level spectrum.

**Comment by cromewar** [comment]
They already have a "welcome path" on their hackathons targeting newcomers, let's see what they have prepared for the next one!!

**Reply by n4n0b1t3** [reply]
I have to apologize, I rarely use tools like discord, it's complete information overload for me (with too much syntax, and too little semantic for my taste). Actually, after I wrote this post I checked the discord channel anyway and stumbled upon a whole hackathon section [blush]. Let's see if we can get the most active people which are finishing let's say within the next two-three months in this discussion group into one hackathon. Like a team? Would be fun.
